### PR TITLE
Add root file type icon

### PR DESCRIPTION
**Pull request by javfg**
This PR adds the file icon for [root](https://root.cern.ch/) files.

It would be great if this can be put in the ODS along with the other icons for files from FontAwesome. We've made it so the SVG is very similar to the rest of the file type icons. But if you prefer another approach, please let us know!

### CODE REVIEW

**Comment by kulmann** [comment]
Did you create the icon yourself? Can you add a kind of license comment into the svg file?

**Comment by javfg** [comment]
> Did you create the icon yourself? Can you add a kind of license comment into the svg file?

It was created by @elizavetaRa, I just made it black and remove inkscape metadata. I've added a MIT license XML entry. Let me know if that works.

**Comment by javfg** [comment]
Pinging @kulmann. Changes are ready. Thanks for the suggestions!

**Comment by javfg** [comment]
> Please also add an author(s) comment to the svg (github handles). Otherwise the license statement is hard to follow.

@kulmann do you mean adding @elizavetaRa in the `Co-authored-by`?

The rest of changes are applied too.

File: src/assets/icons/resource-type-root-fill.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512">
+   <!-- MIT License: https://opensource.org/licenses/MIT -->
```
Comment (kulmann) [inline src/assets/icons/resource-type-root-fill.svg:2]: sorry, I meant adding another comment here in the svg file, linking yours and @elizavetaRa github profiles as authors. just something like:
`<!-- author: https://github.com/elizavetaRa -->`

Comment (javfg) [reply src/assets/icons/resource-type-root-fill.svg:2]: Ah, that makes sense! I've just added that.

**Review by kulmann — APPROVED** [review]
